.azure-perf-ci.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - Documentation/*
+    - /*.md
+
+pr: none
```

Comment:
Can we have this run as optional in PRs? Via comments? Or stuff needs to still be figured out for this build to be public?